### DISCUSSION TITLE
Leaf3 node_group_key. Was expected but not set error

### DISCUSSION THREAD

**Discussion by Jayceehayden** [aristanetworks/avd#1683] [Q&A] [answered]
Hey, I'm trying to create a new node type in avd Ive tried multiple ways to try to fix this however for the node specific  the error keep saying node key type was expected but not set trying to figure out whats causing. I'm pretty sure this has been solved before.

**Comment by ClausHolbechArista** [comment, marked as answer]
Duplicate of issue #1684
